gateway/manage.py

```diff
@@ -14,7 +14,17 @@
 
 def main():
     """Run administrative tasks."""
-    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "main.settings")
+    application_mode = os.environ.get("APPLICATION_MODE", "api")
+    if application_mode == "api":
+        DJANGO_SETTINGS_MODULE = "main.settings_api"
+    elif application_mode == "scheduler":
+        DJANGO_SETTINGS_MODULE = "main.settings_scheduler"
+    elif application_mode == "admin_panel":
+        DJANGO_SETTINGS_MODULE = "main.settings_admin_panel"
+    else:
+        DJANGO_SETTINGS_MODULE = "main.settings_base"
+
+    os.environ.setdefault("DJANGO_SETTINGS_MODULE", DJANGO_SETTINGS_MODULE)
```

Comment:
`manage.py` needs to load settings too because it's what we are using to run commands in the `scheduler`.